x-pack/test/fleet_api_integration/apis/epm/bulk_upgrade.ts

```diff
@@ -51,8 +51,7 @@ export default function (providerContext: FtrProviderContext) {
         expect(body.response.length).equal(1);
         expect(body.response[0].name).equal('multiple_versions');
         const entry = body.response[0] as BulkInstallPackageInfo;
-        expect(entry.oldVersion).equal('0.1.0');
-        expect(entry.newVersion).equal('0.3.0');
+        expect(entry.version).equal('0.3.0');
```

Comment:
I'm fine with removing `oldVersion` as the SecuritySolution app doesn't use it but the intent behind it was to be able to identify when an install for the first time occurs vs an upgrade of a package.

Reply:
Interesting, I removed this since it was logic that was specific to the now-removed `upgradePackage` method and to add it to `installPackage` would mean additional overhead. Maybe we can leverage the new `status` field to provide more granularity about what type of install happened (currently it's a blanket `install`), even if we don't return the actual "old version" info.